doctr/io/elements.py

```diff
@@ -253,6 +258,89 @@ def synthesize(self, **kwargs) -> np.ndarray:
 
         return synthesize_page(self.export(), **kwargs)
 
+    def export_as_xml(self, file_title: str = 'docTR - XML export (hOCR)', skip_rotated_boxes: bool = False) \
```

Comment:
For now it's better to simply raise a `TypeError` if we detect that the geometry is made of rotated bbxo